experiments/examples/experiment.rs

```diff
@@ -5,7 +5,13 @@
 use anyhow::Result;
 use experiments::{AppContext, Experiments};
 fn main() -> Result<()> {
-    let exp = Experiments::new(AppContext::default(), "../target/mydb", None);
-    exp.get_experiments();
+    use env_logger::Env;
+
+    env_logger::from_env(Env::default().default_filter_or("warn")).init();
```

Comment:
could you please add a comment here about the usage for the ENV_LOGGER, the docs are in https://docs.rs/env_logger/0.7.1/env_logger/ but it would be useful to have a quick note about how to tweak the log level right in here. I think it would be useful once we have more trace logging in the SDK